.taskcluster.yml

```diff
@@ -75,6 +75,15 @@ tasks:
                   else:
                       $if: 'tasks_for in ["cron", "action"]'
                       then: '${push.branch}'
+          base_sha:
+              $if: 'tasks_for == "github-push"'
+              then: '${event.before}'
+              else:
+                  $if: 'tasks_for == "github-pull-request"'
+                  then: '${event.pull_request.base.sha}'
+                  else:
+                      $if: 'tasks_for in ["cron", "action"]'
+                      then: '${push.revision}'
```

Comment:
aside: we really need to get these definitions into some shared place 😅

Reply:
Yeah, I totally agree! All along that PR, I was like: "we need a new schema for `.taskcluster.yml` so that we stop having that much boilerplate". I'm having some ideas!